api/reports/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from rest_framework.authtoken.models import Token
+
+from datetime import date
+
+
+class Report(models.Model):
+    """
+    Report model definition
+    """
+    report_type_choices = (
+        ('AN', 'annual'),
```

Comment:
Defining constants for 'AN', 'MO' and 'WE' can help avoid bugs. I.e lets assume how do you filter monthly reports?